docs/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+---
+hide:
+  - toc
+---
+
+# Release notes
+
+{!
+    include-markdown "../README.md"
+    start="## Release notes"
```

Comment:
I assume that this setup is temporary, so we can integrate with the current README, and then down the line we'd break the README down into separate files?

Reply:
Yes, each section can probably be a separate file I think.